geocodingsearchv7/client.go

```diff
@@ -34,6 +38,8 @@ type Client struct {
 	Geocoding *GeocodingService
 	// ReverseGeocoding service
 	ReverseGeocoding *ReverseGeocodingService
+	// BatchGeocoder service
+	BatchGeocoder *BatchGeocoderService
```

Comment:
Relevant if previous comment about name inconsistency is relevant:
```suggestion
	// BatchGeocoding service
	BatchGeocoding *BatchGeocodingService
```